index.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
+var os = require('os');
 var path = require('path');
 
-var homedir = require('homedir-polyfill');
 var isAbsolute = require('is-absolute');
 var removeTrailingSep = require('remove-trailing-separator');
```

Comment:
@sttk Can we use `path.normalize` instead of `remove-trailing-separator`? It seems that the function will collapse multiple `path.sep` so we should be able to do `path.normalize(os.homedir() + path.sep)`, right?